apps/form-builder/src/app/api/Forms/route.ts

```diff
@@ -0,0 +1,11 @@
+import dbConnect from '@/utils/db-connect';
+import { getAllForms } from '@/server/actions/forms';
+import { NextResponse } from 'next/server';
+
+//import something for getForms()
+
+export async function GET (): Promise<NextResponse> {
+  await dbConnect();
```

Comment:
My bad on an incorrect story, this line can be removed, it is inside of the `getAllForms` function